server/middlewares/authentication.js

```diff
@@ -0,0 +1,17 @@
+const jwt = require('jsonwebtoken')
+
+const authentication = function (req, res, next) {
+  try {
+    const token = req.headers.token
+    const decoded = jwt.verify(token, process.env.SECRET)
```

Comment:
ini pindahin ke helpers
```suggestion
const verify = require('../helpers/jwt').verify

const authentication = function (req, res, next) {
  try {
    const token = req.headers.token
    const decoded = verify(token, process.env.SECRET)
```

Reply:
done